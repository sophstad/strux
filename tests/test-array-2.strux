int main() {
	num[4] myArray = [5.4, 4.1, 1.5, 24.1];
  	print(myArray[2]);

  	myArray[2] = 3.5;
  	print(myArray[2]);
    myArray.show();
  	return 0;
}
