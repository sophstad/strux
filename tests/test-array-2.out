1.500000
3.500000